modules/update-upgrading-cli.adoc

```diff
@@ -58,6 +58,23 @@ $ oc get clusterversion -o json|jq ".items[0].spec"
 For production clusters, you must subscribe to a `stable-\*` or `fast-*` channel.
 ====
 
+. Optional: To change the default `upstream` parameter value in cluster version run:
++
+[source,terminal]
+----
+$ oc edit clusterversion -o yaml
```

Comment:
I would suggest to use the `oc patch` command line to change it.

`oc patch clusterversion/version --patch '{"spec":{"upstream":"https://example.com/graph"}}' --type=merge`

Reply:
ack